apps/pipelines/nodes/nodes.py

```diff
@@ -453,6 +453,17 @@ class RouterNode(RouterMixin, Passthrough, HistoryMixin):
         json_schema_extra=UiSchema(widget=Widgets.expandable_text),
     )
 
+    @model_validator(mode="after")
+    def check_prompt_variables(self) -> Self:
+        context = {
+            "prompt": self.prompt,
+        }
+        try:
+            validate_prompt_variables(context=context, prompt_key="prompt", known_vars=set(PromptVars.values))
```

Comment:
Since we cannot select source material, tools or a collection, I think we should support the `participant_data` key only

```suggestion
            validate_prompt_variables(context=context, prompt_key="prompt", known_vars=set([PromptVars.PARTICIPANT_DATA]))
```